tests/test_pqserver.py

```diff
@@ -386,63 +367,74 @@ def test_flexible_metadata(client):
     assert "foo" in result
     assert "bar" in result
     assert "baz" in result
-    assert result["baz"] == ["float", "float key", None, None]
+    assert result["baz2"] == ["float", "float key 2", None, None]
+    assert "foo2" in result
+    assert "bar2" in result
+    assert "baz2" in result
 
     response = client.post(
-        "/consdb/flex/latiss/exposure/obs/2024032100002",
-        json={"values": {"foo": True, "bar": 42, "baz": 3.14159}},
+        "/consdb/flex/latiss/exposure/obs/7024040300451",
+        json={"values": {"foo2": True, "bar2": 42, "baz2": 3.14159}},
```

Comment:
I feel like I am missing some detail, or just skimmed over it on accident. 
Why all the '2's

Reply:
Ideally, between each test, the db would be trashed/downed and brought back up. Is that why the 2s, to make them unique? I would be happy to be wrong here.

Reply:
Yes, for sure the database should be trashed and rebuilt for each test. And this is what happens. But the database is also pre-populated with data, including some pre-existing "foo", "bar", and "baz" keys. Hence the 2's, to avoid a namespace collision.